### ISSUE TITLE
bug in module display

### ISSUE THREAD

**Issue by limingv5** [npm/newww#870]
I published a module:
`https://www.npmjs.com/package/zs-freeway`

Then I unpublished it by --force, but it still can be searched, and it displays in error.

**Comment by aredridel** [comment]
Agreed, this is suboptimal. We'll see about getting this squared away.
